.github/workflows/eamxx-gh-ci-standalone.yml

```diff
@@ -0,0 +1,69 @@
+name: gh-standalone
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      # first, yes to these
+      - '.github/workflows/eamxx-gh-ci-standalone.yml'
+      - 'cime_config/**'
+      - 'components/eam/**'
```

Comment:
I guess we need something from eam, namely a few files for p3/shoc and the rrtmgp submodule, correct? If so, let's limit ourselves to those changes. I don't want standalone to trigger for every single change in eam...